android/app/src/main/res/values-es/strings.xml

```diff
@@ -1103,7 +1103,7 @@
 	<string name="type.cuisine.malagasy">Cocina malgache</string>
 	<string name="type.cuisine.malaysian">Cocina malaya</string>
 	<string name="type.cuisine.mediterranean">Cocina mediterránea</string>
-	<string name="type.cuisine.mexican">Cocina mexicana</string>
+	<string name="type.cuisine.mexican">Cocina mejicana</string>
```

Comment:
Mexicans prefers the spell with "x", and since both "x" and "j" are accepted by RAE, IMO "x" should be preferred.

Reply:
@muralito: We have both specific spellings, for Spanish (Spain) and for Spanish (Mexican):

```
    es = Cocina mejicana
    es-MX = Cocina mexicana
```

Reply:
To fix search, please check data/categories.txt

Reply:
thanks. i will check...

Reply:
Thanks @muralito for your detailed (and convincing) arguments.

I will keep then only the `es = Cocina mexicana` string.